app/main.py

```diff
@@ -1 +1,42 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def die(self) -> None:
+        if self.health <= 0:
+            Animal.alive.remove(self)
+
+    def __repr__(self) -> str:
+        return (f"{{"
+                f"Name: {self.name},"
+                f" Health: {self.health},"
+                f" Hidden: {self.hidden}"
+                f"}}"
+                )
+
+
+class Herbivore(Animal):
+
+    def hide(self) -> None:
+        self.hidden = not self.hidden
+
+
+class Carnivore(Animal):
+
+    @staticmethod
+    def bite(herbivore: "Herbivore") -> None:
```

Comment:
Replace this type annotation to indicate class without quotes
```suggestion
    def bite(herbivore: Herbivore) -> None:
```
In case you face errors, use `from __future__ import annotations` import.